fil-proofs-param/src/bin/paramcache.rs

```diff
@@ -182,6 +212,30 @@ fn generate_params_porep(sector_size: u64, api_version: ApiVersion) {
                     .get(&sector_size)
                     .expect("unknown sector size"),
             ),
+            update_partitions: UpdateProofPartitions::from(partition_count(nodes_count)),
+            h_select: HSelect::from(hs(nodes_count)[0]),
```

Comment:
This expression is used a lot, maybe add sth like  `HSelect::from_count(nodes_count)` as a method.